keyboards/handwired/lagrange/info.json

```
@@ -0,0 +1,32 @@
+{
+  "keyboard_name": "Lagrange",
+  "url": "https://github.com/dpapavas/lagrange-keyboard",
+  "maintainer": "dpapavas",
+  "width": 17,
+  "height": 8,
+  "layouts": {
+    "LAYOUT_lagrange": {
```

Needs to be an exact match for the name used in `lagrange.h`.

```suggestion
    "LAYOUT": {
```